.konflux/releaseplanadmissions/serverless-operator-135-1350-prod.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: appstudio.redhat.com/v1alpha1
 kind: ReleasePlanAdmission
 metadata:
-  name: serverless-operator-135
+  name: serverless-operator-135-1350
```

Comment:
/lgtm

I am fine with both, but IMO `serverless-operator-1350` would be clearer here. But we can also do this as a follow up
/hold in case you want to address it

Reply:
I think it's fine?

Reply:
better than what we have now, so we can improve later

/unhold

Reply:
changing that is not that simple as the app name includes the major/minor version